00-study-plan/study-plan.md

```diff
@@ -1,37 +1,37 @@
 
  ### Path B
-Path B will allow you to dedicate most of your first two months solely to internship. We do recommend keeping up with career development sessions as they occur since preparation for career development sessions is optional and you do have the 5 hours on Thursdays set aside. We also recommend you complete one mock interview on a topic from the classroom phase early on to familiarize yourself with the format. 
+Path B will allow you to dedicate most of your first two months solely to internship. We do recommend keeping up with career development sessions as they occur since preparation for career development sessions is optional and you do have facilitated time for unit coursework set aside. We also recommend you complete one mock interview on a topic from the classroom phase early on to familiarize yourself with the format. 
 
 Path B will introduce all topics to you by the end of internship and allow you to complete all CS Fundamentals Graduation requirements with a week long buffer by the deadline. You will review each new topic and accompanying problem set by the review session. By graduation, you may feel that you need additional review of the CS Fun topics but should have a solid foundation to conduct your review independently.
 
-If you are able to read through the new topic material (without completing the problem set questions) during your first two months of internship , we would highly recommend you do so that the topics are not completely unfamiliar when you start intensive work on CS Fundamentals material.
+If you are able to read through the new topic material (without completing the problem set questions) during your first two months of internship, we would highly recommend you do so so that the topics are not completely unfamiliar when you start intensive work on CS Fundamentals material.
 
  ### Path C
  Choose your own completion dates by filling out and submitting [your personal study plan](https://docs.google.com/document/d/1uCUKu9sZLUSbxsUVobnf8d2EaudXIQxqFpSRhhYE2jU/edit?usp=sharing)
 
  ## Need to Change your Study Plan?
-We expect that most interns will need to adjust their initial study plan. If you are altering any milestones we ask that you update your study plan and fill out the milestone extension form, to give the Thursdays at Ada team awareness of your progress:
+We expect that most interns will need to adjust their initial study plan. If you are altering any milestones we ask that you update your study plan and fill out the milestone extension form, to give the instructional team awareness of your progress:
 [Milestone Extension Form](https://docs.google.com/forms/d/e/1FAIpQLSeBxKIJUhkQ9li3ZOmsSnqlQ38xnBhLEYO8Bnh2CkJkf8i3Ww/viewform?usp=sf_link)
 
  ## Final Deadline 
  
- All Thursdays at Ada work must be submitted to Learn by Friday, July 21st at 11:59 PM. 
\ No newline at end of file
+ All unit work must be submitted to Learn by Friday, July 21st at 11:59 PM. 
```

Comment:
Does the date here need to be updated? I was wondering if it might be helpful for finding the places that need updating if it was switched to "ADD-DEADLINE-HERE" like in the resumes file.

Reply:
It does need to be updated, but it's definitely on my list of things to get updated along with other changes to discuss for the study plan, so I don't think a placeholder is required. Really, the only reason I changed the other location is so that it wouldn't match any more when I searched for "Thur" 😁